utils/urlProvider.ts

```diff
@@ -0,0 +1,5 @@
+const urlProvider = {
+  rpcUrl: "https://gobi-testnet.horizenlabs.io/ethv1",
```

Comment:
At some point, make this customizable

Reply:
Good point! I added it to the comment for the test RPC ticket https://horizenlabs.atlassian.net/browse/TOOLS-2323 so I can keep track!